storops/connection/connector.py

```diff
@@ -54,7 +54,8 @@ class UnityRESTConnector(object):
     }
 
     def __init__(self, host, port=443, user='admin', password='',
-                 verify=False, retries=None, cache_interval=0):
+                 verify=False, retries=None, cache_interval=0,
+                 connect_timeout=10):
```

Comment:
Why this change?

Reply:
It is for the replication feature. When DR happens, Unity could be totally unable to connect. The `connect_timeout` gives storops a quick fail and storops will raise a specific exception. Then OpenStack driver knows the Unity is down.

Reply:
I am not sure whether 10 seconds are enough in some extreme cases. Does it impact all the requests to Unity?

Reply:
Actually, there are several timeout. `connect_timeout` is only one of them and it is only for the time of setting up the connection, doesn't include data reading and writing time.

Yes, all requests to Unity are impacted.

Reply:
If the connection cannot be connected in 10 seconds, storops will timeout?
Would it be too short? 
Please think about 30 seconds or even 60 seconds. I believe normally it is not hit.

Reply:
OK. Set it to 30 sec.